eventActions/highlightActions.js

```diff
@@ -70,7 +71,7 @@ class highlightActions {
 					}
 				}
 				if(contains){
-					const user = client.users.get(currentId);
+					const user = await client.users.cache.get(currentId);
```

Comment:
Do we need `async/await` for this? Correct me if I'm wrong, but I don't think `get` is an async function.

Reply:
You were correct. The await is redundant as `client.users.cache` is a map and as such is not an async call. The await has been removed everywhere it was used.